CODEOWNERS

```diff
@@ -1,5 +1,6 @@
 # Changes to any file in this repo require approval from a member of the InfluxData ui-team
 
 *   @influxdata/ui-team
+*   @influxdata/ecommerce
 /src/writeData/subscriptions/ @influxdata/data-acquisition
 /src/types/subscriptions.ts @influxdata/data-acquisition
```

Comment:
can you add a newline to the end of this file please? 

https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline